sdk/textanalytics/Azure.AI.TextAnalytics/tests/Infrastructure/TextAnalyticsClientLiveTestBase.cs

```diff
@@ -25,7 +25,7 @@ public class TextAnalyticsClientLiveTestBase : RecordedTestBase<TextAnalyticsTes
         protected TextAnalyticsClientOptions.ServiceVersion ServiceVersion { get; }
 
         public TextAnalyticsClientLiveTestBase(bool isAsync, TextAnalyticsClientOptions.ServiceVersion serviceVersion)
-            : base(isAsync, RecordedTestMode.Live)
+            : base(isAsync)
```

Comment:
I actually recommend making that argument something like:

```c#
..., null /* RecordingMode.Record /* remove "null /*" to force recording /*)
```

Like we do in most files. Makes really easy to just remove `null /*` and you can force recording when necessary. Automatic re-recording only happens for recording mismatches.

Reply:
I undid the change for now as it was accidental. Will consider this in the future. Thanks!